File: src/main/java/dev/svenehrke/demo/inbound/web/hono-web-api-shared-consts.ts
// Java-HONO: hand-kept in sync with HonoWebApiSharedConsts.java (the two mutation
// endpoint path templates — see PersonActionResource.java). Can't be generated
// like JTSPersonRouteName / JTSPersonEventName because the TS side consumes the
// actual runtime string values, not just the type.
export const HonoWebApiConsts = {
	PERSON: `/person/{id}`,
	DELETE: `/delete`,
};

File: src/main/java/dev/svenehrke/demo/inbound/web/HonoWebApiSharedConsts.java
package dev.svenehrke.demo.inbound.web;

public interface HonoWebApiSharedConsts {

    interface HonoWebApiConsts {
        String PERSON = "/person/{id}";
        String DELETE = "/delete";
    }

    interface EvtBackendEvents {
        String PERSON_UPDATED = "person-updated";
    }
}
